src/util/graph.h

```diff
@@ -252,6 +252,16 @@ class grapht
   std::vector<node_indext>
   get_reachable(const std::vector<node_indext> &src, bool forwards) const;
 
+  std::vector<node_indext> depth_limited_search(
```

Comment:
Nit pick, just placing it here for I can't comment on commit messages on GitHub: typo in the commit message (s/depht/depth/).